go.mod

```diff
@@ -46,6 +46,7 @@ require (
 	golang.org/x/text v0.3.2 // indirect
 	golang.org/x/tools v0.0.0-20200415000939-92398ad77b89 // indirect
 	google.golang.org/grpc v1.26.0
+	gopkg.in/natefinch/lumberjack.v2 v2.0.0
```

Comment:
MIT license, good to go
Just did a little more research, golang doesn't have a very standard way to do rolling logs, this library seems to be a reasonable choice. Also noticed this lib was mentioned in Zap's doc: https://github.com/uber-go/zap/blob/master/FAQ.md#does-zap-support-log-rotation.